ci-operator/step-registry/olmv1/performance/olmv1-performance-commands.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
```

Comment:
This line is leading the script to crash due to unbound variables, fixed in https://github.com/cloud-bulldozer/e2e-benchmarking/pull/772